packages/components/tests/dummy/app/templates/components/button.hbs

```diff
@@ -111,6 +112,9 @@
   {{! prettier-ignore-end }}
   <p class="dummy-paragraph">Renders to:</p>
   <Hds::Button @text="Copy to clipboard" @icon="clipboard-copy" />
+  <p class="dummy-paragraph"><strong>🚨 IMPORTANT:</strong>
+    <code class="dummy-code">tertiary</code>
+    buttons are required to have either a leading or trailing icon to be accessible.</p>
```

Comment:
Question: Code looks good to me. Should we add an icon to the stickersheet before merge to match ^? Also in the stickersheet "States"

<img width="1318" alt="image" src="https://user-images.githubusercontent.com/1372946/158626822-3c24c0ee-4224-40a5-a2ff-c3298d03e5bc.png">

Reply:
This is something Josh Klekamp will fix in the documentation and sticketsheet in Figma. Once done I'll update (again) the image.

Reply:
@amyrlam https://github.com/hashicorp/design-system/pull/90